health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -13,21 +13,28 @@ import { QuestionContext } from "../../components/CreateQuestionContext";
 
-let temp_data=[]
+let temp_data = []
 
 const CreateChecklist = () => {
   const { t } = useTranslation();
   const tenantId = Digit.ULBService.getCurrentTenantId();
   const searchParams = new URLSearchParams(location.search);
   const [showToast, setShowToast] = useState(null);
   const checklistType = searchParams.get("checklistType");
+  let clt = searchParams.get("checklistType");
+  const checklistTypeLocal = (!clt.startsWith("HCM_CHECKLIST_TYPE_")) ? "HCM_CHECKLIST_TYPE_" + clt : clt;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)